modular_skyrat/modules/customization/modules/mob/living/carbon/human/species/akula.dm

```diff
@@ -1,6 +1,20 @@
+/// How long the akula will stay wet for, AKA how long their species buff will last without being sustained by h2o
+#define DRY_UP_TIME 10 MINUTES
```

Comment:
I feel like this should be handled by the wet stacks like you had for when they get wet, but when they're dry instead, when they reach like, just about 0, basically (I don't think 0 itself will work because it might delete itself by then, but you get what I mean). It's also going to make it far more reactive when they're, say, set on fire, for instance.

Reply:
I don't know what you mean.